Core/include/Acts/Seeding/Seedfinder.ipp

```diff
@@ -366,10 +366,12 @@ void Seedfinder<external_spacepoint_t, platform_t>::createSeedsForGroup(
         // if deltaTheta larger than allowed scattering for calculated pT, skip
         if (m_config.enableCutsForSortedSP) {
           if (deltaCotTheta2 - error2 > p2scatterSigma) {
-            if (cotThetaB - lt.cotTheta < 0) {
-              break;
+            if (m_config.skipPreviousTopSP) {
```

Comment:
Following Robert's comment, can you check if this if should contain both the breaking condition and the update of the t0 parameter?